cmd/version.go

```diff
@@ -49,8 +50,20 @@ func newVersionCmd(homedir, getIstioVersion string) *cobra.Command {
 			}
 
 			logger.Infof("getistio version: %s\nactive istioctl: %s\n", getIstioVersion, cur.ToString())
+			k8sCLient, err := util.GetK8sClient()
+			hasK8sErr := false
+			if err != nil {
+				logger.Infof("no active Kubernetes clusters found\n")
+				hasK8sErr = true
+			} else {
+				_, err := k8sCLient.ServerVersion()
+				if err != nil {
+					logger.Infof("cannot retrieve Kubernetes cluster server information\n")
+					hasK8sErr = true
+				}
+			}
```

Comment:
I think hasK8sErr is not necessary
```suggestion
			k8sCLient, err := util.GetK8sClient()
			if err != nil {
				logger.Infof("no active Kubernetes clusters found\n")
			} else {
				_, err = k8sCLient.ServerVersion()
				if err != nil {
					logger.Infof("cannot retrieve Kubernetes cluster server information\n")
				}
			}
```